File: apps/problems/Convert-Celsius-to-Fahrenheit/boilerplate-full/function.java

    import java.io.*;
    import java.util.*;
    
    public class Main {
        
        ##USER_CODE_HERE##
    
        public static void main(String[] args) {
            String filePath = "/dev/problems/Convert-Celsius-to-Fahrenheit/tests/inputs/##INPUT_FILE_INDEX##.txt"; 
            List<String> lines = readLinesFromFile(filePath);
            float celsius = Float.parseFloat(lines.get(0).trim());
            float result = celsiusToFahrenheit(celsius);
            System.out.println(result);
        }
        public static List<String> readLinesFromFile(String filePath) {
            List<String> lines = new ArrayList<>();
            try (BufferedReader br = new BufferedReader(new FileReader(filePath))) {
                String line;
                while ((line = br.readLine()) != null) {
                    lines.add(line);
                }
            } catch (IOException e) {
                e.printStackTrace();
            }
            return lines;
        }
    }

File: apps/problems/Convert-Celsius-to-Fahrenheit/boilerplate-full/function.js
##USER_CODE_HERE##
    
    const input = require('fs').readFileSync('/dev/problems/Convert-Celsius-to-Fahrenheit/tests/inputs/##INPUT_FILE_INDEX##.txt', 'utf8').trim().split('\n').join(' ').split(' ');
    const celsius = parseInt(input.shift());
    const result = celsiusToFahrenheit(celsius);
    console.log(result);
        